### ISSUE TITLE
Better error reporting for missing jQuery

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#185]
We could detect a case when jQuery is missing and put a human readable error instead of

> Uncaught TypeError: Cannot read property 'fn' of undefined

**Comment by Tade0** [comment]
I see this one is more or less fixed, but before it can be merged the branch needs to get rebased and also a manual test is required.

**Comment by mlewand** [comment]
There was a missing exclaimation mark, and with that `$.fn` would always evaluate to `true` when jQuery is in place. Fixed that and I've removed the code as this fix will only work for the built version. Which is fine, as for dev version the error says `Uncaught ReferenceError: jQuery is not defined` which should be understood by devs.

Technically we could provide error reporting for dev version too, but it would require us to put a _shim_ for Quail's dependency, where we would check if jQuery was found, and if not exception would be thrown. However it's not worth our effort, since `bender-amd` plugin does not allow to pass shims (functions).
